src/main.rs

```diff
@@ -206,6 +206,8 @@ async fn main() -> Result<()> {
                         get_optional_argument_value(sub_m, "bbs_public_key"),
                         get_optional_argument_value(sub_m, "signign_key"),
```

Comment:
is that a typo for the parameter name?

Reply:
it was a typo and it has been fixed now :)